lib/mix/test/mix/task_test.exs

```diff
@@ -17,43 +17,47 @@ defmodule Mix.TaskTest do
       Mix.Task.run("unknown")
     end
 
-    assert_raise Mix.NoTaskError, "The task \"helli\" could not be found. Did you mean \"hello\"?", fn ->
-      Mix.Task.run("helli")
-    end
+    assert_raise Mix.NoTaskError,
+                 "The task \"helli\" could not be found. Did you mean \"hello\"?",
```

Comment:
Please do what highlighted in "Example 3" of https://github.com/elixir-lang/elixir/issues/6643 here and in the similar situations below.